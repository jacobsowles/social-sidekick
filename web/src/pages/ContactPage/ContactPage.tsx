import React, { FunctionComponent } from 'react';

import ContentBox from '@components/ContentBox';
import PageHeader from '@components/PageHeader';
import './ContactPage.scss';

const ContactPage: FunctionComponent = () => {
  return (
    <div className="contact-page">
      <ContentBox>
        <PageHeader title="Contact" />
        <p>Page content</p>
      </ContentBox>
    </div>
  );
};

export default ContactPage;
