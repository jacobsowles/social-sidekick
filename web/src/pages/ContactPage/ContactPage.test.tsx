import React from 'react';

import { init, shallow, shallowTopLevelElement, toJson } from '@tests/component-test-base';
import ContactPage from '@pages/ContactPage';

describe('ContactPage', () => {
  init(() => <ContactPage />, 'div');

  it('should render the component without crashing', () => {
    toJson(shallow()).should.matchSnapshot();
  });

  it('should only include the default class name', () => {
    shallowTopLevelElement()
      .prop('className')
      .should.equal('contact-page');
  });
});
