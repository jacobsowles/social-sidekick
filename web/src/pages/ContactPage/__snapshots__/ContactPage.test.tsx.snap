// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage should render the component without crashing 1`] = `
<div
  className="contact-page"
>
  <ContentBox>
    <PageHeader
      title="Contact"
    />
    <p>
      Questions? Comments? Feel free to get in touch via 
      <a
        href="mailto:jacob@jacobsowles.com"
      >
        email
      </a>
       or 
      <a
        href="https://github.com/jacobsowles/social-sidekick"
      >
        GitHub
      </a>
      .
    </p>
  </ContentBox>
</div>
`;
